rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/custom_subscriber_info.hpp

```diff
@@ -102,15 +102,15 @@ class SubListener : public eprosima::fastrtps::SubscriberListener
   }
 
   void
-  data_taken()
+  data_taken(eprosima::fastrtps::Subscriber * sub)
```

Comment:
Please add docstring for the new input param.

Reply:
I think this change came from ros2/master.